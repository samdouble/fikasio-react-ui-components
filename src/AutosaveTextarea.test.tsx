import React from 'react';
import renderer from 'react-test-renderer';
import { AutosaveTextarea } from '.';

describe('AutosaveTextarea', () => {
  it('Renders correctly', () => {
    const componentJSON = renderer.create(
      <AutosaveTextarea
        onSave={() => undefined}
      />,
    ).toJSON();
    expect(componentJSON).toMatchSnapshot();
  });
});
