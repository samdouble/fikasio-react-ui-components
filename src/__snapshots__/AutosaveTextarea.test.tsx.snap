// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AutosaveTextarea Renders correctly 1`] = `
<div
  className="fikasio-textarea fikasio-theme-dark"
  contentEditable={true}
  dangerouslySetInnerHTML={
    Object {
      "__html": "",
    }
  }
  onBlur={[Function]}
  onChange={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onInput={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  style={
    Object {
      "whiteSpace": "pre",
    }
  }
/>
`;
